ix-dev/community/windmill/templates/test_values/basic-values.yaml

```diff
@@ -0,0 +1,70 @@
+windmill:
+  postgres_image_selector: postgres_16_image
+  web_host: localhost
+  db_password: changeme
+  base_url: ":80"
```

Comment:
this shouldn't be needed here.
You should match the port with the given host port.